exchanges/binance/binance_test.go

```diff
@@ -3487,3 +3487,19 @@ func TestGetOpenInterest(t *testing.T) {
 	})
 	assert.ErrorIs(t, err, asset.ErrNotSupported)
 }
+
+func TestGetCurrencyTradeURL(t *testing.T) {
+	t.Parallel()
+	testexch.UpdatePairsOnce(t, b)
+	for _, a := range b.GetAssetTypes(false) {
+		pairs, err := b.CurrencyPairs.GetPairs(a, false)
+		if len(pairs) == 0 {
+			continue
+		}
```

Comment:
I think it would be better to require it to not be empty. Should be supporting all asset types. Also can we check error first? and can you please change that for all tests?

Reply:
I did it to avoid modifying other functions. I've done it now 🤙  It already was for all supporting asset types